### DISCUSSION TITLE
MacPorts Necessary for Git Installation

### DISCUSSION THREAD

**Discussion by walthyneman** [community/community#55132] [New to GitHub]
I'm 99.99% sure the answer is 'yes' or it not be noted in the "Download for MacOS" web-page as a step, but I thought I'd ask anyway... I'm on an older MacBookPro (Big Sur : ver 11.7.6). When I try to install Xcode (needed for the MacPorts step) I'm unable to since Xcode now requires OS v13+. 

On a tutorial "install git" youtube video < https://www.youtube.com/watch?v=ZM3I16Z-lxI >, I noticed the MacPorts step was not listed in the downloads page (I think the vid is 2 years old).

Can I continue setting up gitHub without MacPorts installed on my Mac or will this doom me forever in broken gitHubHell ?

W.

ps. I anticipate the solution besides bying a newer Mac will be to install an older version of git/gitHub ??

**Comment by queenofcorgis** [comment]
Hey there! I'd recommend checking out [this page ](https://git-scm.com/download/mac) - the binary installer option should be the easiest followed by Homebrew.
